.circleci/config.yml

```diff
@@ -496,6 +496,7 @@ jobs:
           name: Copy documents
           command: |
             git config --local user.name "AWS"
+            git fetch --depth 1 origin gh-pages
```

Comment:
For release doc, I think we should build the doc based on the release branch and force push to gh-pages?